reverse-proxy/traefik/docker-compose.traefik.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.3"
+services:
+  plausible:
+    labels:
+      traefik.enable: "true"
+      traefik.http.routers.plausible.rule: "Host(`example.com`)" # change to your domain name
```

Comment:
Is it possible for this line to access `plausible-conf.env` so we can use the `BASE_URL` configuration param here?

Reply:
Good idea!

It [should be possible](https://docs.docker.com/compose/compose-file/#variable-substitution) to use an env variable in the label by simply changing it to ``"Host(`${BASE_URL}`)"``. That requires the environment variable to be set for the `docker-compose` process though.

I believe the easiest way to do this is to either specify the environment variable in [a file called `.env`](https://docs.docker.com/compose/environment-variables/#the-env-file) or by specifying [`docker-compose --env-file=plausible-conf.env up`](https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files) (although the documentation for the latter seems a bit dodgy?).

It should probably be verified by running `docker-compose config` with both options, but I am unfortunately unable to do so right now.